vhdbuilder/scripts/windows/automate_release_notes.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+set -euxo pipefail
+
+source vhdbuilder/scripts/windows/automate_helpers.sh
+
+az login --identity
+
+echo "Build Id is $1"
+ 
+build_id=$1
+
+set +x
+github_access_token=$2
+set -x
+
+github_user_name=$3
+
+az pipelines runs artifact download --run-id $build_id --artifact-name publishing-info-2019-containerd --path .
```

Comment:
It assumes that all image versions must be same. It is rare with different image versions but it does exist when building vhds cross 00:00AM.

Reply:
this is just for branch naming.

The `sigImageVersion` in Line 348, `main.go` above is read from the original artifact.

If the version for 2019-containerd is say 230808, for 2022-containerd is 230809, then `getReleaseNotesWindows` for sku 2019-containerd will use `230808` and it will use `230809` for 2022-containerd. The branch name will be the same `230808`. 

That's why I initially used default/custom build date for branch naming.

I don't see a problem here sorry. That `getReleaseNotesWindows` function is called for every sku, just like Linux did.

Reply:
Reference: https://github.com/Azure/AgentBaker/pull/3434/files L27-L28, `automate_version_bump.sh`. Again, this is only used for defining the name of the branch. `sigImageVersion` is read from the original artifact and used as the name for the release notes `releaseNotesFileOut` (L353, `main.go`)